File: python/1038.py
n = input()

[codigo, quantidade] = n.split()

cod = int(codigo)
quant = int(quantidade)
cachorroq = float(quant * 4.00)
x_salada = float(quant * 4.50)
x_bacon = float(quant * 5.00)
torrada = float(quant * 2.00)
refri = float(quant * 1.50)


if cod == 1:
    print("Total: R$ %.2f" % cachorroq)
if cod == 2:
    print("Total: R$ %.2f" % x_salada)
if cod == 3:
    print("Total: R$ %.2f" % x_bacon)
if cod == 4:
    print("Total: R$ %.2f" % torrada)
if cod == 5:
    print("Total: R$ %.2f" % refri)

File: c/1038.c
#include <stdio.h>

int main()
{
  int a, b;
  scanf("%d"
        "%d",
        &a, &b);

  float cachorroq = (b * 4.00);
  float x_salada = (b * 4.50);
  float x_bacon = (b * 5.00);
  float torrada = (b * 2.00);
  float refri = (b * 1.50);

  if (a == 1)
  {
    printf("Total: R$ %.2f\n", cachorroq);
  }
  else
  {
    if (a == 2)
    {
      printf("Total: R$ %.2f\n", x_salada);
    }
    else
    {
      if (a == 3)
      {
        printf("Total: R$ %.2f\n", x_bacon);
      }
      else
      {
        if (a == 4)
        {
          printf("Total: R$ %.2f\n", torrada);
        }
        else
        {
          if (a == 5)
          {
            printf("Total: R$ %.2f\n", refri);
          }
        }
      }
    }
  }
  return 0;
}